server/sonar-server/src/test/java/org/sonar/server/issue/IssueServiceMediumTest.java

```diff
@@ -603,7 +603,7 @@ private void newSourceLine(ComponentDto file, int line, String scmAuthor) {
 
   private void newUser(String login) {
     MockUserSession.set().setLogin("admin").setGlobalPermissions(GlobalPermissions.SYSTEM_ADMIN);
```

Comment:
Please remove this call to the user session as thanks to your refactoring it's no more mandatory to have any permission to create a new user !